freetextresponse/private/view.html

```diff
@@ -6,6 +6,9 @@ <h2 class="problem-header">{self.display_name}</h2>
         <textarea class="student_answer" style="height:150px" rows="20" cols="70">{self.student_answer}</textarea>
         <span class="status {indicator_visibility_class}" aria-describedby="student_answer"></span>
     </div>
+    <div class="submission-received">
+        {submitted_message}
```

Comment:
What does this look like in loco?

'cause from the code it looks like this is rendered as text inside of a `<div>` tag which isn't optimal...